_episodes/40.md

```diff
@@ -0,0 +1,479 @@
+While this provides a nice migration from Hive, many are looking to replace this
+rather cumbersome catalog with something that's lightweight and doesn't rely on
+Hive's Thrift API. Two popular catalogs outside of these are the 
+[JDBC catalog](https://iceberg.apache.org/docs/latest/jdbc)
+and the 
+[REST catalog](https://github.com/apache/iceberg/pull/4348).
+
+There are two PRs in progress to add these catalogs to Trino:
+
+* [Trino PR 11772: Support JDBC catalog in Iceberg connector](https://github.com/trinodb/trino/pull/11772)
+* [Trino PR 13294: Add Iceberg RESTSessionCatalog Implementation](https://github.com/trinodb/trino/pull/13294)
+
+### Branching, tagging, and auditing, oh my!
+
+Another set of features that are coming in Iceberg is the ability to use 
+[refs to alias your snapshots](https://github.com/apache/iceberg/pull/5364).
+This would enable branching and tagging behavior similar to git and treating
+the snapshot as a commit. This is yet another way that will simplify moving
+between known states of the data in Iceberg.
+
+Another interesting feature that refs feed into, is the 
+[ability to audit the data in Iceberg](https://tabular.io/blog/integrated-audits).
+Auditing allows you to push a soft commit by making a snapshot available, but 
+is not initially published to the primary table. This is achieved using Spark
+and setting the `spark.wap.id`. This enables interesting patterns like 
+[Write-Audit-Publish (WAP) pattern](https://www.dremio.com/subsurface/write-audit-publish-pattern-via-apache-iceberg/),
+where you first write the data, audit it using a data quality tool like Great 
+Expectations, and publish the data to be visible from the main table. Currently,
+auditing has to use the cherry-pick operation to publish. Eventually this will
+become more streamlined with branching and tagging.
+
+### Puffin file format
+
+The [Puffin file format](https://iceberg.apache.org/puffin-spec) is an
+alternative to [Parquet](https://parquet.apache.org/) and 
+[ORC](https://orc.apache.org/) that store information such as indexes and 
+statistics about data managed in an Iceberg table that cannot be stored directly
+within the Iceberg manifest. A Puffin file contains arbitrary pieces of 
+information (here called “blobs”), along with metadata necessary to interpret 
+them.
+
+This format [was suggested](https://www.mail-archive.com/dev@iceberg.apache.org/msg03593.html)
+by our own [Piotr Findeisen @findepi](https://github.com/findepi) and was
+proposed to address a performance issue noted when using Trino on Iceberg. The
+Puffin format will be a great extension for those using Iceberg tables.
+
+### pyIceberg
+
+Another exciting development is the 
+[pyIceberg library](https://github.com/apache/iceberg/tree/master/python) that
+enables users to read their data directly from Iceberg into pythonic data
+structures.
+
+### Trino connector updates
```

Comment:
> cut down to just talk about the features added in the Trino releases since the last TCB about Iceberg

That's what these are lol I didn't include any feature that existed when we did the last TCB episode in April 2021. Also these are all new since my last blog on Iceberg came out. So all of this we should touch on.

> not really relevant with what specific release

Fair enough. I'll keep the link to the PRs then and they can dig into it if they want.

Reply:
I do want to keep the explanation for time travel though. That is a little confusing and it will be helpful to know what they can use in which version.